test/auto_parallel/static_reshard_api_cross_mesh.py

```diff
@@ -160,8 +161,10 @@ def test_reshard_mesh(self):
                 'pd_op.relu',
                 'pd_op.matmul',
                 'pd_op.add',
-                'pd_op.send_v2',
-                'pd_op.recv_v2',
+                'pd_op.share_data_',
+                'pd_op.broadcast',
+                'pd_op.full',
+                'pd_op.broadcast',
```

Comment:
does share_data_ and full affect the performance?

Reply:
Performance verification in progress